code/modules/mob/dead/new_player/sprite_accessories_Citadel.dm

```diff
@@ -950,34 +956,44 @@
 /datum/sprite_accessory/mam_ears/guilmon
 	name = "Guilmon"
 	icon_state = "guilmon"
-	icon = 'icons/mob/mam_bodyparts.dmi'
+
+/datum/sprite_accessory/snout/guilmon
+	name = "Guilmon"
+	icon_state = "guilmon"
 
 /datum/sprite_accessory/mam_tails/shark/datashark
 	name = "DataShark"
 	icon_state = "datashark"
 	color_src = 0
-//	ckeys_allowed = list("rubyflamewing")
+	ckeys_allowed = list("rubyflamewing")
 
 /datum/sprite_accessory/mam_tails_animated/shark/datashark
 	name = "DataShark"
 	icon_state = "datashark"
 	color_src = 0
 
-/*
-//Till I get my snowflake only ckey lock, these are locked-locked :D
+/datum/sprite_accessory/mam_body_markings/shark/datashark
+	name = "DataShark"
+	icon_state = "datashark"
+	color_src = MUTCOLORS2
+	ckeys_allowed = list("rubyflamewing")
```

Comment:
lmao dude

did you really have to hardcode in your fursona

Reply:
like really

of all the things your codebase needed, why did your fursona need hardcoded in when you could of just greyscaled it so everyone can use the part

Reply:
One, it isn't mine, mine's further below. 

Two, because we like having unique things that sets us apart from each other.

Reply:
Have you considered making your character's backstory, personality, and character flaws set them apart instead?

Reply:
It's nice to have a visual representation of the character. Especially since it isn't just a chatroom.

Reply:
Then what makes you special enough to get to have your own unique sprite when everyone else is stuck with the standard kit?

Reply:
Because I coded it. c:

Reply:
I could PR giving my ckey a free pulse rifle when I spawn and that doesn't mean I get it. Try again.

Reply:
cosmetic addition != a gameplay advantage. 

if anything, in a game where appearances matter, it's a downside.

Reply:
No, its an upside. You literally cant be impersonated by a traitor now. Try again.

Reply:
sure can. tails are hidden by hardsuits. voice changer + ID, no one will be the wiser.

Lings get these too upon absorb

Reply:
ITT: goof doesnt understand what a fursona is while simultaneously forgetting the existence of many different game mechanics

Reply:
Do I look like I know what a fursona is? I just want to print a picture of a god damn clean PR.